CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ _When adding new entries to the changelog, please include issue/PR numbers where
 - Fixed a bug where Kart would require `user.name` and `user.email` to be set, even when `GIT_AUTHOR_EMAIL` and similar variables were set in the environment. [#812](https://github.com/koordinates/kart/issues/812)
 - Update native dependencies. [#820](https://github.com/koordinates/kart/pull/820)
 - Drop CI testing of Debian Stretch/oldoldstable. [#820](https://github.com/koordinates/kart/pull/820)
+- Allow kart to import non-spatial tables from PostgreSQL databases which don't have postGIS extension installed. [#728](https://github.com/koordinates/kart/issues/728)
```

Comment:
You should merge / rebase with master - git may not flag this as a conflict, but this should bullet point now be in 0.12.4 (UNRELEASED)